frame/benchmarking/src/lib.rs

```diff
@@ -1804,3 +1818,57 @@ macro_rules! list_benchmark {
 		$list.push(pallet_benchmarks)
 	)
 }
+
+/// Callback for `define_benchmarks` to call `list_benchmark`.
+
+#[macro_export]
+macro_rules! cb_list_benchmarks {
+    // anchor
+	( $list:ident, $extra:ident, [ $name:path, $( $location:tt )* ] ) => {
+        list_benchmark!( $list, $extra, $name, $( $location )* );
+	};
+    // recursion tail
+	( $list:ident, $extra:ident, [ $name:path, $( $location:tt )* ] $([ $names:path, $( $locations:tt )* ])+ ) => {
+        cb_list_benchmarks!( $list, $extra, [ $name, $( $location )* ] );
+        cb_list_benchmarks!( $list, $extra, $([ $names, $( $locations )* ])+ );
+    }
+}
+
+/// Defines pallet configs that `add_benchmarks` and `list_benchmarks` use.
+/// Should be preferred instead of having a repetitive list of configs
+/// in `add_benchmark` and `list_benchmark`.
+
```

Comment:
are these newlines intentional?

Reply:
Hm they are also above other macros. I have to admit that I'm not sure why.  
I will remove them.